poppy/instrument.py

```diff
@@ -474,8 +474,12 @@ def _getOpticalSystem(self,fft_oversample=2, detector_oversample = None, fov_arc
 
 
         #---- apply pupil intensity and OPD to the optical model
-        optsys.addPupil(name='Entrance Pupil', optic=pupil_optic, transmission=full_pupil_path, opd=full_opd_path, opdunits='micron', rotation=self._rotation)
+        optsys.addPupil(name='Entrance Pupil', optic=pupil_optic, transmission=full_pupil_path, opd=full_opd_path, rotation=self._rotation)
```

Comment:
Well, the two options are:

1. Require a webbpsf-data update with units on OPD maps, have the default OPD units be meters
2. Make the default OPD units microns, but do not require a webbpsf-data update

I thought the benefit to consistency (meters everywhere) was worth it, but we could change the default too